docs/docs/classic-ml/tracking/quickstart/index.mdx

```diff
@@ -200,6 +199,39 @@ with mlflow.start_run():
     )
 ```
 
+:::note Console Output
+When you run the above code, you will see console output that looks something like this:
+
+```text
+INFO:     127.0.0.1:56522 - "PATCH /api/2.0/mlflow/logged-models/m-8b08caa5365f4b10ac8e3fe2c8bf001c HTTP/1.1" 200 OK
+2025/09/09 17:22:20 ERROR mlflow.webhooks.delivery: Failed to deliver webhook for event registered_model.created: FileStore does not support list_webhooks_by_event
+Traceback (most recent call last):
```

Comment:
Could we truncate this display to just show the fact that there are a lot of lines here, followed by the NotImplementedError line? (just to focus on the readability of this note)

Reply:
Quick question... My environment is getting error for using `ruff` `0.12.12` rather than `0.12.10`. Would it be appropriate to add a commit or create an issue to bump it to `0.12.12` (Sept 4th) or even `0.13.0` (today) in `pyproject.toml`?